inst/extdata/scripts/run_audit.R

```diff
@@ -0,0 +1,32 @@
+logger::log_threshold(Sys.getenv("LOG_LEVEL", "INFO"))
+
+schema_tempdir <- tempdir()
+workflow.pacta.webapp::prepare_schema_files(
+  directory = schema_tempdir
+)
+
+raw_params <- commandArgs(trailingOnly = TRUE)
+params <- pacta.workflow.utils::parse_raw_params(
+  json = raw_params,
+  inheritence_search_paths = c(
+    system.file(
+      "extdata", "parameters",
+      package = "workflow.pacta.webapp"
+    ),
+    system.file(
+      "extdata", "parameters",
+      package = "workflow.pacta"
+    )
+  ),
+  schema_file = file.path(schema_tempdir, "PACTAParameters.json"),
+  raw_schema_file = file.path(schema_tempdir, "rawParameters.json"),
+  force_array = c("portfolio", "files")
+)
+
+workflow.pacta.webapp::run_webapp_workflow(
+  params = params,
+  raw_params = raw_params,
+  run_audit = TRUE,
+  run_analysis = FALSE,
+  run_reporting = FALSE
+)
```

Comment:
I don't think this will work, `run_webapp_worfklow` doesn't have a `run_audit` argument

(Note that https://github.com/RMI-PACTA/workflow.pacta/pull/107 adds this to the `run_pacta` function instead)

Reply:
good catch. Thanks.

Reply:
https://github.com/RMI-PACTA/workflow.pacta.webapp/pull/32/commits/3dfd9b96ab3d93728a5c9a58a277ae2eb887b076